airflow/providers/google/suite/hooks/sheets.py

```diff
@@ -115,7 +115,7 @@ def get_values(
             .execute(num_retries=self.num_retries)
         )
 
-        return response["values"]
+        return response.get('values', [])
```

Comment:
Why empty list rather than None?

Reply:
Used an empty list to respect the function's signature, but `None` could also be valid if we're willing to change the return type to `Optional[list]`.

Reply:
To me it makes more sense to return what the API returns. If the API returns nothing so should we.